osu.Game.Rulesets.Osu/Mods/OsuModRateChanger.cs

```diff
@@ -7,6 +7,6 @@ namespace osu.Game.Rulesets.Osu.Mods
 {
     public class OsuModRateChanger : ModRateChanger
     {
-        public override double ScoreMultiplier => 0;
+        public override double ScoreMultiplier => SpeedChange.Value;
```

Comment:
what is this change?  just leave the multiplier at 1 for now or copy whatever the existing rate change mods do please

Reply:
Just a test, I shouldn't have pushed it.